framework/src/controller/validator/formats.js

```diff
@@ -367,8 +366,8 @@ const validationFormats = {
 		type: 'number',
 		validate: value => {
 			const { TOTAL_AMOUNT } = global.constants;
-			if (new Bignum(value).isPositive()) {
-				return new Bignum(value).isLessThanOrEqualTo(TOTAL_AMOUNT);
+			if (new BigNum(value).isPositive()) {
+				return new BigNum(value).isLessThanOrEqualTo(TOTAL_AMOUNT);
```

Comment:
Wouldn't be `lessThanOrEqualTo`?

```suggestion
				return new BigNum(value).lessThanOrEqualTo(TOTAL_AMOUNT);
```

Reply:
Good catch!